src/Couchbase.Lite.Shared/API/Database/Database.cs

```diff
@@ -540,14 +563,12 @@ public void CreateIndex([@NotNull]string name, [@NotNull]IIndex index)
         /// <exception cref="InvalidOperationException">Thrown if this method is called after the database is closed</exception>
         public void Delete()
         {
-            StopExpirePurgeTimer(); 
```

Comment:
Looks like this was mistakenly removed?

Reply:
Because it's handled in Close/Dispose method.